src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -157,7 +158,8 @@ public static IEnumerable<ClassMirror> GetAllTypes(Core core)
                 // if not used in the language: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6752
                 return core.ClassTable.ClassNodes.
                     Where(x => !CoreUtils.StartsWithSingleUnderscore(x.Name)
-                    && x.Name != DSDefinitions.Keyword.PointerReserved).
+                    && x.Name != DSDefinitions.Keyword.PointerReserved
+                    && !x.HasDSDerivedClass).
```

Comment:
If I hide the base class using `IsVisibleInLibrary(false)` autocomplete will hide the members too.